sci-libs/minc-toolkit/minc-toolkit-9999.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI="5"
+
+inherit cmake-utils git-r3
+
+DESCRIPTION="Metaproject uniting all the available MINC tools"
+HOMEPAGE="https://github.com/BIC-MNI/minc-toolkit"
+SRC_URI=""
+EGIT_REPO_URI="git://github.com/BIC-MNI/minc-toolkit.git"
+
+LICENSE="BSD"
+SLOT="0"
+IUSE=""
+KEYWORDS=""
+
+COMMON_DEP="media-libs/freeglut
+	x11-libs/libXmu
+	x11-libs/libXi"
+
+DEPEND="dev-lang/perl
+	sys-devel/bison
+	sys-devel/flex
+	${COMMON_DEP}"
+
+RDEPEND="sci-libs/hdf5
+	sci-libs/netcdf
+	${COMMON_DEP}"
+
+src_configure() {
+	local mycmakeargs=( -DUSE_SYSTEM_HDF5=1 )
+	cmake-utils_src_configure
```

Comment:
Seriously, you shouldn't use software for scientific purposes you don't know. fftw3f is the floating point library, while fftw3 is the double precision lib.
Further, just check the cmake modules in https://github.com/BIC-MNI/minc-toolkit/tree/master/cmake-modules to see from which destination the sources are fetched. This will point you to the right packages if compared with the output of `eix $packagename`.